fetch.bs

```diff
@@ -1121,6 +1111,46 @@ or "<code>worker</code>".
  <li><p>Return <var>newRequest</var>.
 </ol>
 
+<p>To <dfn export for=request id=concept-request-upload-body>upload body</dfn> of a
+<a for=/>request</a> <var>request</var>, run these steps:
+
+<ol>
+ <li>Let <var>body</var> be <var>request</var>'s <a for=request>body</a>.
+
+ <li><p>If <var>body</var> is null, then <a>queue a fetch task</a> on <var>request</var> to
+ <a>process request end-of-body</a> for <var>request</var> and abort these steps.
+
+ <li>
+  <p>Let <var>read</var> be the result of <a lt="read a chunk" for=ReadableStream>reading a
+  chunk</a> from <var>body</var>'s <a for=body>stream</a>.
+
+  <ul>
+   <li>
+    <p>When <var>read</var> is fulfilled with an object whose <code>done</code>
+    property is false and whose <code>value</code> property is a
+    <code>Uint8Array</code> object, run these substeps:
```

Comment:
Oh, I see, I was focusing on the Uint8Array issue, but the "object whose done property is false" part is the imprecise part. In particular, what if `obj.done` or `obj.value` is a throwing getter? Let me push a commit to fix.